### ISSUE TITLE
Porting to Android

### ISSUE THREAD

**Issue by oct-test** [libimobiledevice/libimobiledevice#298]
Is it possible to port this library to **Android**? If yes, which other libraries I need to port in order to access iOS device's data from Android application connected via OTG (on-the-go) cable?

**Comment by songhaonangit** [comment]
You can have a try.

**Comment by cczhr** [comment]
yes, I used this open source project to make an android project that uses otg to connect to iphone https://github.com/cczhr/OTGLocation
